proto/google/events/cloud/audit/v1/LogEntryData.json

```diff
@@ -1,387 +1,426 @@
 {
-    "$id": "google.events.cloud.audit.v1",
-    "$schema": "http://json-schema.org/schema",
-    "title": "Cloud Audit Log LogEntry v1",
-    "name": "LogEntryData",
-    "description": "This event is triggered when a new audit log entry is written.",
-    "type": "object",
-    "goPackage": "auditv1",
-    "definitions": {
-        "MonitoredResource": {
```

Comment:
I'm not sure about embedding MonitoredResource etc within the CAL-specific schema. If another data proto uses MonitoredResource, would that have another copy of it, preventing them from being used interoperably?

Have you considered creating a "common" schema that can be imported into this one, assuming such a concept exists in JSON schema?

Reply:
This PR doesn't add it. I agree we should extract it when there are multiple instances.

Yes, I think a common schema would be great and we should add that once we split the json schemas to their own folder.